xfields/ibs/_equilibrium_emittance.py

```diff
@@ -232,6 +261,30 @@ def compute_emittance_evolution(
     T_z : list of float
         Longitudinal IBS growth rates computed over all the time steps.
     """
+    # Returns
+    # -------
+    # xtrack.TwissTable
+    #     The convergence calculations results. The table contains the following
+    #     columns, as time-step by time-step quantities:
+    #         - time: time values at which quantities are computed, in [s]
+    #         - gemitt_x: horizontal geometric emittance values, in [m]
+    #         - gemitt_y: vertical geometric emittance values, in [m]
+    #         - gemitt_zeta: longitudinal geometric emittance values, in [m]
+    #         - Tx: horizontal IBS growth rate, in [s^-1]
+    #         - Ty: vertical IBS growth rate, in [s^-1]
+    #         - Tz: longitudinal IBS growth rate, in [s^-1]
+    #     The table also contains the following global quantities:
+    #         - damping_constants_s: radiation damping constants per second used for the calculations
+    #         - partition_numbers: damping partition numbers used for the calculations
+    #         - eq_gemitt_x: final horizontal equilibrium geometric emittance converged to, in [m]
+    #         - eq_gemitt_y: final vertical equilibrium geometric emittance converged to, in [m]
+    #         - eq_gemitt_zeta: final longitudinal equilibrium geometric emittance converged to, in [m]
+    #         - sr_eq_gemitt_x: horizontal equilibrium geometric emittance from synchrotron radiation used, in [m]
+    #         - sr_eq_gemitt_y: vertical equilibrium geometric emittance from synchrotron radiation used, in [m]
+    #         - eq_nemitt_zeta: longitudinal equilibrium normalized emittance from synchrotron radiation used, in [m]
```

Comment:
Commented out for now. This could replace the `Returns` section of the docstring when we make the move to return an `xtrack.Table` directly. Just like the docstring for the `TwissTable` I have added info separately on the columns and "global quantities" we store in the object.